new/LIT7150Miracle.xml

```diff
@@ -0,0 +1,130 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7150Miracle" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>                
+                <title xml:lang="gez" xml:id="t1">Miracle of St Mary on the monk Isacc who Prayed for St.Mary's Presence</title>                
+                <editor role="generalEditor" key="AB"/>
+                <editor key="GS"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                       
+                            This file is licensed under the Creative Commons
+                            Attribution-ShareAlike 4.0. 
+                    </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <abstract>
+                <p>
+                    Yǝsḥaq, a devoted monk under Bishop Gabriel of Alexandria, dedicated himself
+                    to fervent nightly prayers before the Virgin Mary's icon for seven years, bowing
```

Comment:
@nafisa-valieva notes
Guesh, are you sure that Gabriel was a bishop and not a patriarch? I think it would be very helpful to have Pers id here instead of the name.

Reply:
@nafisa-valieva notes
Thank you very much, Guesh! Do I understand correctly that this miracle was never edited or translated or discussed? I would imagine it was translated from Arabic and there must be something on it, no?